### PR TITLE
[cli] Fix `vc logout` command when using Team scope

### PR DESCRIPTION
**Pull request by TooTallNate**
Fixes logout command not working when switched to a Team scope:

```
$ vc login
$ vc switch $some_team
$ vc logout
Failed during logout
```

### CODE REVIEW

File: packages/cli/src/types.ts [hunk 1]
```diff
@@ -16,7 +16,7 @@ export interface JSONObject {
 }
 
 export interface AuthConfig {
-  token: string;
+  token?: string;
```
Comment (TooTallNate) [inline packages/cli/src/types.ts:19]: VS Code was complaining (though the version of TypeScript we use to build doesn't seem to mind).

File: packages/cli/src/commands/logout.ts [hunk 2]
```diff
@@ -1,6 +1,5 @@
 import chalk from 'chalk';
 import logo from '../util/output/logo';
-// @ts-ignore
```
Comment (TooTallNate) [inline packages/cli/src/commands/logout.ts:3]: Apparently not necessary 🤷

**Review by AndyBitz — APPROVED** [review]

File: packages/cli/src/commands/logout.ts (hunk 2 above)
Comment (markozxuu) [reply packages/cli/src/commands/logout.ts:3]: Right, the comment `@ts-ignore` is intended to ignore TS errors (**not recommend**), If what I code is well, not necessary that comment : -)

File: packages/cli/src/types.ts (hunk 1 above)
Comment (styfle) [reply packages/cli/src/types.ts:19]: @TooTallNate You can tell VS Code to use the same version as the build by clicking the version in the bottom right of VS Code > Select TypeScript Version > Use Workspace Version.

https://stackoverflow.com/a/59076553/266535

**Review by styfle — APPROVED** [review]

File: packages/cli/src/types.ts (hunk 1 above)
Comment (TooTallNate) [reply packages/cli/src/types.ts:19]: Oh, nice TIL.

**Pull request merged**